### PR TITLE
Fix  for docs in RPA.Assistant

### PR DESCRIPTION
**Pull request by bogdancondurache**

### CODE REVIEW

**Review by mikahanninen — APPROVED** [review]
LGTM

File: docs/source/releasenotes.rst [hunk 1]
```diff
@@ -96,7 +96,6 @@ Release notes
   compatibility mode. (robot example: https://github.com/robocorp/example-ie-mode-edge)
 - Library **RPA.SAP**: Fix error in dependency import order in the underlying
   `SapGuiLibrary`.
->>>>>>> master
```
Comment (cmin764) [inline docs/source/releasenotes.rst:99]: This already got solved in the latest _master_.

File: pyproject.toml [hunk 2]
```diff
@@ -13,7 +13,7 @@ rpaframework-google = "^7.0.0"
 rpaframework-openai = "^1.0.1"
 rpaframework-recognition = "^5.0.1"
 rpaframework-windows = "^7.0.3"
-rpaframework-assistant = "^1.0.4"
+rpaframework-assistant = "^1.0.5"
```
Comment (cmin764) [inline pyproject.toml:16]: Better to run a `poetry update -vvv` (or `inv install.update`) directly in _master_ (alongside this change) once you merge and release the new assistant package, so you get it in the docs straight from PyPI.

(as PyPI releases usually work from _master_ post-merge)

**Review by cmin764 — APPROVED** [review]
LGTM! (left two nits)

File: pyproject.toml (hunk 2 above)
Comment (xylix) [reply pyproject.toml:16]: we have been building the pypi releases by hand for rpaframework-assistant, but valid point about the workflow.

Comment (bogdancondurache) [reply pyproject.toml:16]: makes sense, I'll remove this change from the PR and we can do it afterwards then

Comment (bogdancondurache) [reply pyproject.toml:16]:
```suggestion
rpaframework-assistant = "^1.0.4"
```

File: docs/source/releasenotes.rst (hunk 1 above)
Comment (bogdancondurache) [reply docs/source/releasenotes.rst:99]: shouldn't be an issue with it being here then 😄

**Pull request merged**